Homework 19

d-beige opened the pull request.

d-beige committed:
Update Homework17.java

d-beige committed:
Homework 19 complete

Completed homework 19 assignment!

Berichicko commented:
Please finish the homework!You got this problem, please correct it. 

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/1789126c-f00f-48d3-996e-49ea0551764a)

HINT

![Test status_en](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/28cf40f7-c19e-4ea0-bb41-430c95cffd90)

For example

![Test failed_en](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/f75f08c3-6ba7-46e5-b949-8bf23cfdd89d)

d-beige committed:
Recompleted Homework 19

Previous submit was functioning but the playlist was already deleted. Made a change to the code that fits dynamic playlist names.

Berichicko commented:
Hello,

I have checked your homework 19, and I wanted to let you know that you're doing great! Keep up the excellent work!
Best wishes from the HW team (Alex, [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io))!

![804deb64dc406a20a75da05d91a1a69e](https://user-images.githubusercontent.com/104761482/235961479-9afb2841-b657-46c9-99f8-2402f933b843.jpg)

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/3125febf-d99f-4158-858e-03eb5afe7ad9)
